src/common/snippets/src/op/subgraph.cpp

```diff
@@ -257,16 +257,16 @@ auto Subgraph::wrap_node_as_subgraph(const std::shared_ptr<ov::Node>& node) -> s
 }
 
 void Subgraph::fill_empty_output_names(const Output<Node>& target_output_node, const Output<Node>& replacement_output_node) {
-    OPENVINO_SUPPRESS_DEPRECATED_START
+    //OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
I believe this a part of another PR

Reply:
Yeah, I think, it was late at night :)
Fixed.